third_party/terraform/website/docs/r/compute_router_nat.html.markdown

```diff
@@ -19,21 +19,21 @@ A simple NAT configuration: enable NAT for all Subnetworks associated with
 the Network associated with the given Router.
 
 ```hcl
-resource "google_compute_network" "network" {
+resource "google_compute_network" "default" {
```

Comment:
pretty nit but let's avoid `default` as a network name. Even though this is the Terraform uri and not the GCP name, we don't want to conflate this with the project default network.

Reply:
I'll update this if I'm back in the file, but not worth waiting for another magician run.